### PR TITLE
BHBC-1893: Update the Composition and SRB Survey Template

### PR DESCRIPTION
**Pull request by KjartanE**
# Overview

## Links to Jira tickets

- https://quartech.atlassian.net/browse/BHBC-1893
- https://quartech.atlassian.net/browse/BHBC-1880

## Description of relevant changes

- {List all relevant changes, in particular anything that will help the reviewers test/verify this PR}

## PR Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### Code

- [ ] New files/classes/functions have appropriately descriptive names and comment blocks to describe their use/behaviour
- [x] I have avoided duplicating code when possible, moving re-usable pieces into functions
- [x] I have avoided hard-coding values where possible and moved any re-usable constants to a constants file
- [x] My code is as flat as possible (avoids deeply nested if/else blocks, promise chains, etc)
- [ ] My code changes account for null/undefined values and handle errors appropriately
- [ ] My code uses types/interfaces to help describe values/parameters/etc, help ensure type safety, and improve readability

### Style

- [ ] My code follows the established style conventions
- [ ] My code uses native material-ui components/icons/conventions when possible

### Documentation

- [ ] I have commented my code sufficiently, such that an unfamiliar developer could understand my code
- [ ] I have added/updated README's and related documentation, as needed

### Tests

- [ ] I have added/updated unit tests for any code I've added/updated
- [ ] I have added/updated the Postman requests/tests to account for any API endpoints I've added/updated

### Linting/Formatting

- [ ] I have run the linter and fixed any issues, as needed  
       _See the `lint` commands in package.json_
- [ ] I have run the formatter and fixed any issues, as needed  
       _See the `format` commands in package.json_

### SonarCloud

- [ ] I have addressed all SonarCloud Bugs, Vulnerabilities, Security Hotspots, and Code Smells

### CODE REVIEW

**Review by curtisupshall — DISMISSED** [review]
🐄

**Comment by NickPhura** [comment]
What are we doing with the transformations these days? I know we put them on hold before, is that still the case now? Not sure if it came up at all while working through this ticket with Robin

**Review by anissa-agahchen — DISMISSED** [review]

**Comment by anissa-agahchen** [comment]
* resource page updates (when?)

File: database/src/migrations/template_methodology_species_validations/picklist_variables/v0.2.ts [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+  { name: 'Tracks' },
+  { name: 'Whitewash' },
+  { name: 'Described in Comments' }
+];
+
+const SheepSpecies = [
+  { name: 'Bighorn Sheep' },
+  { name: 'Ovis canadensis' },
+  { name: 'M-OVCA' },
+  { name: "Dall''s Sheep" },
```
Comment (curtisupshall) [inline database/src/migrations/template_methodology_species_validations/picklist_variables/v0.2.ts]: Is a double apostrophe expected?

**Review by curtisupshall — DISMISSED** [review]
👷

**Review by curtisupshall — APPROVED** [review]
🐑

**Pull request merged**